internal/checker/cloudformation/rules/general-security/verify_immutability_for_ecr_image_tags.rego

```diff
@@ -0,0 +1,48 @@
+# METADATA
+# title: "Verify Immutability for ECR Image Tags."
+# description: "This directive mandates the immutability of image tags within the Elastic Container Registry (ECR), fostering stability and reliability in container image management."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecr-repository.html
+# custom:
+#   ID: CB_CFT_39
+#   Severity: Low
+package lib.cloudformation.CB_CFT_39
+import future.keywords.in
+
+resource := "AWS::ECR::Repository"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
update implementation to return properties object
pass[Properties] {
	is_valid
	some resources in input.Resources
	resources.Type == resource
	Properties := resources.Properties
	block.ImageTagMutability == "IMMUTABLE"
}